httpmw/limiter_v1.go

```diff
@@ -0,0 +1,51 @@
+package httpmw
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/go-redis/redis/v8"
+)
+
+// NewRedisLimiter creates redisLimiter.
```

Comment:
I suggest avoiding struct names, so I would update this to `// NewRedisLimiter creates new redis limiter.`.